packages/testkit/tsconfig.json

```diff
@@ -1,28 +1,22 @@
 {
-    "compilerOptions": {
-      "module": "commonjs",
-      "target": "es6",
-      "outDir": "dist",
-      "strict": true,
-      "sourceMap": true,
-      "strictNullChecks": true,
-      "moduleResolution": "node",
-      "importHelpers": true,
-      "esModuleInterop": true,
-      "skipLibCheck": true,
-      "forceConsistentCasingInFileNames": true,
-      "noUnusedLocals": true,
-      "noUnusedParameters": true,
-      "noFallthroughCasesInSwitch": true,
-      "strictBindCallApply": true,
-      "strictFunctionTypes": true,
-      "strictPropertyInitialization": true,
-      "rootDir": ".",
-      "declaration": false,
-      "downlevelIteration": true,
-      "experimentalDecorators": true,
-      "noImplicitOverride": true,
-      "noImplicitReturns": true,
-    },
-    "include": [ "buttons", "inputs", "navigation", "pickers", "popover", "text", "utils", "BaseElement.ts", "index.ts"]
-  }
+  "compilerOptions": {
+    "lib": ["ESNext"],
+    "target": "es6",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
```

Comment:
do you import json files in your testkit? needed?

Reply:
removed